### PR TITLE
Chore(migration commands): added sequelize migration commands #3xgy4jp

### PR DESCRIPTION
**Pull request by bright258**
### What does this do?
This contains migration commands for migrating up and reverting a database model.

### How to test?

- Pull this branch
- Run `npm run migrate:up` to migrate up
- Run `npm run migrate:down` to migrate down

### CODE REVIEW

**Comment by Endowmissy** [comment]
Task linked: [CU-3xgy4jp Migration commands](https://app.clickup.com/t/3xgy4jp)

File: database/migrations/20221201070519-user-migrations.js [hunk 1]
```diff
@@ -45,6 +45,6 @@ module.exports = {
     });
   },
   down: async (queryInterface, Sequelize) => {
-    await queryInterface.dropTable('users');
+    await queryInterface.dropTable('users', {}, null);
```
Comment (Endowmissy) [inline database/migrations/20221201070519-user-migrations.js]: @bright258 why did you update this??

Comment (bright258) [reply database/migrations/20221201070519-user-migrations.js]: I thought adding it will solve the migration issue

Comment (Endowmissy) [reply database/migrations/20221201070519-user-migrations.js]: @bright258 remove it sir

File: package.json [hunk 2]
```diff
@@ -5,8 +5,10 @@
   "main": "server.js",
   "scripts": {
     "start": "node src/server.js",
-    "dev": "nodemon src/server.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "dev": "nodemon src/server.js && npx sequelize-cli db:migrate",
```
Comment (Endowmissy) [inline package.json]: @bright258 migration command should be before the server command

File: database/migrations/20221201070519-user-migrations.js (hunk 1 above)
Comment (bright258) [reply database/migrations/20221201070519-user-migrations.js]: Done

File: package.json (hunk 2 above)
Comment (bright258) [reply package.json]: Done

**Review by Endowmissy — APPROVED** [review]
Good job

**Review by kelue — APPROVED** [review]

**Pull request merged**